src/libs/map.ts

```diff
@@ -0,0 +1,9 @@
+export const getIsInclude = <K, V>(map: Map<K, V>, key: K) => !!map.get(key)
+
+export const mapToArray = <K, V>(map: Map<K, V>) => Array.from(map.keys())
```

Comment:
[MUST] The actual behavior does not match the name `mapToArray`.
```ts
const map = new Map()

map.set('hoge', { name: 'hoge' })
map.set('fuga', { name: 'fuga' })

console.log(mapToArray(map))

/*
Expected value from the name 'mapToArray'
[
  ['hoge', { name: 'hoge' }],
  ['fuga', { name: 'fuga' }],
]

Actual value
['hoge', 'fuga']
*/
```

In the case of the current behavior, I think the name `mapToKeyArray` is appropriate.

Reply:
fixed: https://github.com/kufu/smarthr-ui/pull/408/commits/f75505e7e2c57f1b21169e3ebabc0411a3f69658